src/main/java/com/capitalone/dashboard/model/EvaluationStatus.java

```diff
@@ -0,0 +1,17 @@
+package com.capitalone.dashboard.model;
+
+public enum EvaluationStatus {
+
+    PENDING,
```

Comment:
Will this be used, for example, when the test_result audit report has been created but the testresult-sync collector hasn't run yet?

Reply:
it can be used if the selection of audit reports is based of evaluationStatus